### PR TITLE
separate helpers and filter tests, add dataclass for banner, fix tests

### PR DESCRIPTION
**Pull request by Adam-W1**
Separate helpers and filter tests, add dataclass for banner, fix tests

### CODE REVIEW

File: app/assess/routes.py [hunk 1]
```diff
@@ -117,7 +92,7 @@ def display_sub_criteria(
         "application_id": application_id,
         "fund": fund,
         "comments": comments,
-        "is_flagged": is_flagged,
+        "is_flagged": True if flag else False,
```
Comment (gidsg) [inline app/assess/routes.py:95]: can be simplified
```suggestion
        "is_flagged": bool(flag),
```

File: app/assess/models/round.py [hunk 2]
```diff
@@ -30,10 +30,6 @@ def from_dict(cls, d: dict):
 
     @staticmethod
     def from_json(data: dict):
-        # eligibility_criteria = {}
-        # if "eligibility_criteria" in data:
-        #     for key, value in data["eligibility_criteria"].items():
-        #         eligibility_criteria.update({key: value})
```
Comment (gidsg) [inline app/assess/models/round.py:36]: Can we remove this rather than commenting out?

Comment (Adam-W1) [reply app/assess/models/round.py:36]: that has been removed, it was commented out originally but Im not sure why

Comment (gidsg) [reply app/assess/models/round.py:36]: @Adam-W1 change doesn't seem to have been pushed to remove this?

Comment (Adam-W1) [reply app/assess/models/round.py:36]: The red highlight means it's being removed (Im removing it from main as part of this BAU)

File: app/assess/views/filters.py [hunk 3]
```diff
@@ -8,6 +8,7 @@ def status_to_human(status: str):
         "SUBMITTED": "Submitted",
         "COMPLETED": "Complete",
         "FLAGGED": "Flagged",
+        "STOPPED": "Stopped",
```
Comment (gidsg) [inline app/assess/views/filters.py:11]: Think we could replace this with the built-in Jinja filters either [capitalise](https://tedboy.github.io/jinja2/templ14.html#capitalize) or [title](https://tedboy.github.io/jinja2/templ14.html#title) just awaiting confirmation from design which is correct CC @charlesrt

Comment (gidsg) [reply app/assess/views/filters.py:11]: @Adam-W1 design have confirmed `title` case is the intended behaviour, so please can we remove this and use the built-in `title` filter instead?

Comment (gidsg) [reply app/assess/views/filters.py:11]: @Adam-W1 So I think on the basis of what design have said. `capitalize` is the correct filter to use.

Comment (Adam-W1) [reply app/assess/views/filters.py:11]: I think we still need the custom as the statuses display slightly differently on UI (including removing underscore), For example:
- "NOT_STARTED" -> "Not started"
- "COMPLETED" -> "Assessment completed".

Comment (gidsg) [reply app/assess/views/filters.py:11]: Could try doing that chaining a standard filter in the view e.g. `| replace("_"," ")` 

Or could replace this function with 
```
return status.capitalize().replace('_', ' ')
```
and achieve same result.

Comment (gidsg) [reply app/assess/views/filters.py:11]: Oh I see missed the assessment complete thing

**Review by gidsg — APPROVED** [review]

**Pull request merged**